index.html

```diff
@@ -0,0 +1,161 @@
+    <div class="hero-container">
+      <p class="greeting">"Hello Athlete's"</p>
+      <h1 class="hero-header-text">
+        Victory isn't defined by wins and losses, it's defined by effort
+      </h1>
+      <p class="hero-header-desc">With our team you’ll reach your goals quickly. Enjoy a spacious and well equipped free
+        weight area to meet everyone's needs with the best equipment in the industry. Equipped with the space and
+        equipment for the normal person to competitive athletes or athletic teams.
+      </p>
+      <p class="hero-header-time"><span class="workout-time"><i class="fas fa-clock"></i>24/7</span> (Monday) ~ (Friday)
+      </p>
+    </div>
+  </header>
+
+  <!-- Services Section -->
+  <section class="services">
+    <h2 class="services-header">Main Services</h2>
+    <div class="card-wrapper">
+      <!-- Card-1 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fas fa-duotone fa-dumbbell"></i>
+        </div>
+        <div class="card-header">
+          Body Building
+        </div>
+        <div class="card-body">
+          <p>Build your body with great instactore in the industry. With complete Nutration plane.</p>
+        </div>
+      </div>
+      <!-- Card-2 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fas fa-clock"></i>
+        </div>
+        <div class="card-header">
+          OPEN 7 DAYS A WEEK
+        </div>
+        <div class="card-body">
+          <p>Flexible hours to meet everyone's scheduling needs.</p>
+        </div>
+      </div>
+      <!-- Card-3 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fa-sharp fa-solid fa-hand-sparkles"></i>
+        </div>
+        <div class="card-header">
+          CLEAN, SPACIOUS FACILITIE
+        </div>
+        <div class="card-body">
+          <p>From front lobby and fitness machines to the bathrooms , a clean gym is one of our top priorities.</p>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card-image">
+          <i class="fa-sharp fa-solid fa-hand-sparkles"></i>
+        </div>
+        <div class="card-header">
+          CLEAN, SPACIOUS FACILITIE
+        </div>
+        <div class="card-body">
+          <p>From front lobby and fitness machines to the bathrooms , a clean gym is one of our top priorities.</p>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card-image">
+          <i class="fa-sharp fa-solid fa-hand-sparkles"></i>
+        </div>
+        <div class="card-header">
+          CLEAN, SPACIOUS FACILITIE
+        </div>
+        <div class="card-body">
+          <p>From front lobby and fitness machines to the bathrooms , a clean gym is one of our top priorities.</p>
+        </div>
+      </div>
+    </div>
+    <a href="#" class="btn-join">Join GOLD GYM</a>
+  </section>
+
+  <!-- Feature Speaker -->
+  <section class="services feature-speaker">
```

Comment:
- [ ] Please make sure that the speakers are perfectly in the center. Currently, they are a little bit towards the left.

![image](https://user-images.githubusercontent.com/92215878/190708279-cec23d7d-234b-49b1-8b38-1f6b57dac2e7.png)